packages/components/README.md

```diff
@@ -283,5 +290,55 @@ yarn publish
 ```
 
 This will prompt you for the new version.  Choose a version increment in accordance with [SemVer](https://semver.org/).  This command will
-update the `package.json` file and commit that change.  Then you can do a `git push` to get the update into the remote repository. (Note,
-you could instead use `npm publish`, but you will have to update the `package.json` file manually before using that command.)
+update the `package.json` file and commit that change.  Then you can do a `git push` to get the update into the remote repository.
+
+## Merging changes into master
+
+1. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
+merging at the same time which might result in conflicting package version numbers.
+1. Do one final merge of the `master` branch into your feature branch for `labkey-ui-components`.
+1. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
+1. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
+1. Run the commands to build, test, and publish: `yarn build`, `yarn test`, `yarn publish`.
+1. Push your final set of commits from `labkey-ui-components` to github so that TeamCity can do a final run of the jest tests.
+1. Update any LabKey module `package.json` files where you are using / applying these changes with this final release version
+number (then do the regular `npm install` for that module, build, etc. and push those `package.json` and `package-lock.json` file
+changes to github as well).
+1. Remove any of the alpha package versions from [Artifactory](https://artifactory.labkey.com/artifactory/webapp/#/home)
+that you had published during development for this feature branch.
+1. Check on the [TeamCity](https://teamcity.labkey.org) build status and jest test status.
+1. Merge the pull requests for `labkey-ui-components` and any of your related LabKey modules / repos.
+1. Message the Frontend dev room chat that the merge is complete.
+
+## Making hotfix patches to a release branch
+
+Occasionally we will need to patch a version of the `@labkey/components` package to fix a bug in a previous release of
+LabKey server. There are a few extra steps in this process, described below:
+
+1. Find the `@labkey/components` package version number to patch by looking at the package versions that are used
+ in the `package.json` files of the LabKey module that you plan to update.
```

Comment:
The version number to branch from needs to be the latest bug fix version greater than the version in use by the package where the update is required. That is, if the module uses 0.41.2, but we have already produced 0.41.3 and 0.41.4, you need to use 0.41.4 as the branch starting point since you'll presumably be making another bug fix release and can't use the versions already published.

Reply:
added, thanks